scene/3d/skeleton.cpp

```diff
@@ -538,6 +536,11 @@ void Skeleton::unbind_child_node_from_bone(int p_bone, Node *p_node) {
 	ERR_FAIL_INDEX(p_bone, bones.size());
 
 	uint32_t id = p_node->get_instance_id();
+
+	if (bones.write[p_bone].nodes_bound.find(id) == nullptr) {
+		return; // Doesn't exist in the first place.
+	}
+
```

Comment:
Can you store the element returned and use this to call `erase` using the element directly rather than by value?